installer/roles/provision-mobile-catalog/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+# tasks for provision mobile catalog on an existing openshift cluster
+# NOTE: Ansible Service Broker should be already provisoned in this cluster
+- name: Get current user
+  shell: "oc whoami"
+  register: "oc_whoami_output"
+
+- set_fact:
+    current_user: "{{oc_whoami_output.stdout}}"
+
+- name: Login as system admin
+  shell: "oc login -u system:admin"
+  when: current_user != "system:admin"
+
+- import_role:
+    name: create-mobile-client-crd
+
+# TODO: ideally we want to parse the broker config to add the new registry and update the config, but for now we keep it simple
+- name: update config map
+  block:
+    - name: oc get configmap broker-config
+      shell: oc get configmap broker-config -n {{ansible_service_broker_namespace}} -o yaml > /tmp/broker-config.yaml
+
+    - name: Add aerogear docker registry
+      blockinfile:
+        path: /tmp/broker-config.yaml
+        insertafter: 'registry:'
+        marker: "      # Aerogear registry added by ansible installer"
+        block: |2
+                - type: dockerhub
+                  name: dh
+                  url: https://registry.hub.docker.com
+                  org: aerogearcatalog
+                  tag: 1.0.0
+                  white_list: [.*-apb\$]
```

Comment:
will this be added again if this blocked is already there?

Reply:
yes it would, I will add a check to make sure it doesnt happen